homework-2/app.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+
+const app = express();
+
+const fs = require("fs");
+
+const port = 5000;
+
+app.get("/", (req, res) => {
+  res.send("Greeting from server!");
+});
+
+app.get("/data", (req, res) => {
+  res.send("Welcome to data path");
+});
+
+app.listen(port, () => {
+  console.log(`server listening on ${port}`);
+});
+
+const numbersFile = "./homework-2/text/numbers.txt";
+
+const fileForReadAndDelete = "./homework-2/text/file-for-read-and-delete.txt";
+
+let numbers = "";
+
+fs.writeFile(numbersFile, numbers, (err) => {
+  if (err) throw err;
+  console.log(`${numbersFile} was created`);
+  for (let i = 1; i <= 10; i++) {
+    numbers = numbers + `${i}\n`;
```

Comment:
why don't you prepare numbers to insert, and then just write it to file?
You don't need to use appendFileSync for such trivial task